cdn/agent-loader/prebuilt.js

```diff
@@ -0,0 +1,7 @@
+import { BrowserAgent } from "@newrelic/browser-agent";
+
+const configs = {
+    init: { ajax: { enabled: false }, jserrors: {auto: false, enabled: true} },
+    info: { applicationID: '601303348', licenseKey: 'a60cc46d05', errorBeacon: 'bam.nr-data.net' }
+}
+window.agent = new BrowserAgent(configs)
```

Comment:
What does this do?

Reply:
This was present only for demonstration purposes, needs to be removed.